phastpress.php

```diff
@@ -8,9 +8,56 @@
 License: AGPLv3
 */
 
-
+define('PHASTPRESS_IS_DEV', true);
```

Comment:
Use `'$VERSION$' != '$VER' . 'SION$'` instead. Saves a fragile step in the build.